content/contact/home.pml

```diff
@@ -66,9 +58,12 @@ The next sessions are:
     <ul>
       <li>Sydney: Wed, Mar 23, 8am</li>
       <li>Central Europe: Tue, Mar 22, 10pm</li>
-      <li>US Pacific Time: Tue, Mar 22, 2pm</li>
+      <li>US Pacific Time: Tue, Mar 22, 1pm</li>
```

Comment:
2pm was correct (see https://www.worldtimebuddy.com/?qm=1&lid=100,2950159,2147714,5391959&h=2950159&date=2022-3-22&sln=22-23&hf=0), because they switch from PST to PDT (+1h) on the 14th (see https://www.timeanddate.com/time/zone/usa/los-angeles)

Reply:
This is what I got from timeanddate.com:

UTC (Time Zone)                      Tuesday, 8 March 2022 at 9:00:00 pm   UTC  UTC          
Sydney (Australia - New South Wales) Wednesday, 9 March 2022 at 8:00:00 am AEDT UTC+11 hours 
Los Angeles (USA - California)       Tuesday, 8 March 2022 at 1:00:00 pm   PST  UTC-8 hours    
Berlin (Germany - Berlin)            Tuesday, 8 March 2022 at 10:00:00 pm  CET  UTC+1 hour

Reply:
That's for the 8/9th of March, but the change is for the 22nd/23th.

Reply:
Ah, of course! I'll make the change.